eo-phi-normalizer/src/Language/EO/Phi/Syntax/Abs.hs

```diff
@@ -1,3 +1,15 @@
+-- Copyright 2002-2021 Andreas Abel, Jonas Almström Duregård, Krasimir Angelov, Jean-Philippe Bernardy, Björn Bringert, Johan Broberg, Kyle Butt, Paul Callaghan, Grégoire Détrez, Markus Forsberg, Ola Frid, Peter Gammie, Thomas Hallgren, Pascal Hof, Simon Huber, Patrik Jansson, Kristofer Johannisson, Antti-Juhani Kaijanaho, Andreas Lööw, Justin Meiners, Kent Mein, Ulf Norell, Gabriele Paganelli, Michael Pellauer, Fabian Ruch, and Aarne Ranta.
```

Comment:
So, BNFC has BSD-3 license. This license doesn't impose any restrictions on the generated files. Since `Abs.hs` and `Print.hs` are generated, this license can be replaced with the content of the [license file](https://github.com/objectionary/eo-phi-normalizer/blob/c53ed6a49b5632c4eed7cc347d1fb3bfc1f7361e/eo-phi-normalizer/LICENSE) in the `eo-phi-normalizer` package.